supporter-product-data/src/main/scala/com/gu/model/StageHelpers.scala

```diff
@@ -0,0 +1,18 @@
+package com.gu.model
+
+import com.gu.supporterdata.model.Stage
+import com.gu.supporterdata.model.Stage.{DEV, PROD, UAT}
+
+object StageHelpers {
```

Comment:
I'm not a big fan of "Helpers" as it doesn't really add any meaning, I can see it's renamed from `Stage` so if that's necessary would it be we be more specific?
```suggestion
object StageConstructors {
```